raiden/tests/integration/transfer/test_refundtransfer.py

```diff
@@ -386,8 +426,25 @@ def test_different_view_of_last_bp_during_unlock(
     fee = calculate_fee_for_amount(amount_refund)
     fee_margin = calculate_fee_margin(amount_refund, fee)
     amount_refund_with_fees = amount_refund + fee + fee_margin
+
+    token_network = views.get_token_network_by_token_address(
+        views.state_from_raiden(app0), app0.default_registry.address, token_address,
+    )
+    assert token_network
     payment_status = app0.mediated_transfer_async(
-        token_network_address, amount_refund, TargetAddress(app2.address), identifier_refund
+        token_network_address=token_network_address,
+        amount=amount_refund,
+        target=TargetAddress(app2.address),
+        identifier=identifier_refund,
+        route_states=[
+            RouteState(
```

Comment:
Can't this use the `create_route_state_for_route` helper as well?

Reply:
Nice find, somehow stepped over this while refactoring.